### PR TITLE
US-3522 Added driver-specs unit tests to project

### PR DESCRIPTION
**Pull request by simongranger**

### CODE REVIEW

File: lib/driver.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import commands from './commands/index';
 import _ from 'lodash';
 import B from 'bluebird';
 import { sleep } from 'asyncbox';
+//import { fs } from 'appium-support';
```
Comment (oecheverri) [inline lib/driver.js]: Is this commented out line needed?

Comment (simongranger) [inline lib/driver.js]: nope

File: lib/driver.js [hunk 2]
```diff
@@ -269,8 +281,7 @@ class YouiEngineDriver extends BaseDriver {
       connected = await connectedPromise;
 
       if (!connected && retryCount === (MAX_RETRY_COUNT - 1)) {
-        this.deleteSession();
```
Comment (simongranger) [inline lib/driver.js]: deleteSession was moved to the catch block of createSession

**Comment by jplandry** [comment]
It's all js to me! +0.5

**Comment by oecheverri** [comment]
👍

**Pull request merged**